src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
```

Comment:
you don't need to ad health in constructor

Reply:
agree, health value won't change and will be the same for all animals